### PR TITLE
translate mintable xc-20, permits and other xcm changes

### PR DESCRIPTION
**Pull request by hyd628**
### Description/Original PRs

https://github.com/PureStake/moonbeam-docs/pull/319
https://github.com/PureStake/moonbeam-docs/pull/329
https://github.com/PureStake/moonbeam-docs/pull/334

piggybacking this into this PR too: https://purestake.atlassian.net/browse/MD-759

This does not include the XCM transactor PR. 

### Checklist

- [x] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [x] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [x] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

Please link to any additional corresponding PRs here.

### CODE REVIEW

File: builders/xcm/xc20/overview.md [hunk 1]
```diff
@@ -1,187 +1,141 @@
...
 
-### 编译合约 {: #compile-the-contract }
+![Load the interface in Remix](/images/builders/xcm/xc20/overview/overview-1.png)
 
 当您成功在Remix读取ERC-20接口后，您将需要编译：
 
 1. 点击（从上至下的）第二个**Compile**标签
 
-2. 编译**IERC20.sol**文档
+2. 编译**IER20.sol**文档
```
Comment (eshaben) [inline builders/xcm/xc20/overview.md:120]:
```suggestion
2. 编译**IERC20.sol**文档
```

File: builders/xcm/xc20/xc20.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+description: 学习如何使用预编译的资产Solidity合约通过ERC-20接口与外部迁移至Moonbeam的跨链Token交互。
+---
+
+# 外部XC-20
+
+![Cross-Chain Assets Precompiled Contracts Banner](/images/builders/xcm/xc20/xc20/xc20-banner.png)
+
+## 概览 {: #introduction }
+
+如同[XC-20概况](/builders/xcm/xc20/overview){target=_blank}页面中所述，[XC-20有两种类型](/builders/xcm/xc20/overview#types-of-xc-20s)：外部的和[可铸造的](/builders/xcm/xc20/overview#types-of-xc-20s){target=_blank}。外部和可铸造的XC-20之间最大的不同为外部XC-20代表那些在中继链或是其他平行链中锁定在Moonbeam主权账户的资产。相反地，可铸造XC-20代表那些直接在Moonbeam链上铸造或是销毁的资产但拥有原生XCM互操作性功能。此教程将涵盖外部XC-20类型的资产。
```
Comment (eshaben) [inline builders/xcm/xc20/xc20.md:12]:
```suggestion
如同[XC-20概况](/builders/xcm/xc20/overview){target=_blank}页面中所述，[XC-20有两种类型](/builders/xcm/xc20/overview#types-of-xc-20s){target=_blank}：外部的和[可铸造的](/builders/xcm/xc20/overview#types-of-xc-20s){target=_blank}。外部和可铸造的XC-20之间最大的不同为外部XC-20代表那些在中继链或是其他平行链中锁定在Moonbeam主权账户的资产。相反地，可铸造XC-20代表那些直接在Moonbeam链上铸造或是销毁的资产但拥有原生XCM互操作性功能。此教程将涵盖外部XC-20类型的资产。 
```

File: builders/xcm/xc20/xc20.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+目前每个网络可用的外部XC-20资产列表如下所示：
+
+=== "Moonbeam"
+    |  原始网络  | XC-20表现形式 |                                                           XC-20地址                                                           |
+    |:--------:|:------:|:---------------------------------------------------------------------------------------------------------------------------------:|
+    | Polkadot | xcDOT  | [0xFfFFfFff1FcaCBd218EDc0EbA20Fc2308C778080](https://moonscan.io/token/0xFfFFfFff1FcaCBd218EDc0EbA20Fc2308C778080){target=_blank} |
+    |  Acala   | xcaUSD | [0xfFfFFFFF52C56A9257bB97f4B2b6F7B2D624ecda](https://moonscan.io/token/0xfFfFFFFF52C56A9257bB97f4B2b6F7B2D624ecda){target=_blank} |
+    |  Acala   | xcACA  | [0xffffFFffa922Fef94566104a6e5A35a4fCDDAA9f](https://moonscan.io/token/0xffffFFffa922Fef94566104a6e5A35a4fCDDAA9f){target=_blank} |
+
+     _*You can check each [Asset ID](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbeam.network#/assets){target=_blank} on Polkadot.js Apps_
```
Comment (eshaben) [inline builders/xcm/xc20/xc20.md:33]: this can be translated

File: builders/xcm/xc20/xc20.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    |    Khala     | xcPHA  | [0xffFfFFff8E6b63d9e447B6d4C45BDA8AF9dc9603](https://moonriver.moonscan.io/token/0xffFfFFff8E6b63d9e447B6d4C45BDA8AF9dc9603){target=_blank} |
+    |   Kintsugi   | xcKINT | [0xfffFFFFF83F4f317d3cbF6EC6250AeC3697b3fF2](https://moonriver.moonscan.io/token/0xfffFFFFF83F4f317d3cbF6EC6250AeC3697b3fF2){target=_blank} |
+    |   Kintsugi   | xckBTC | [0xFFFfFfFfF6E528AD57184579beeE00c5d5e646F0](https://moonriver.moonscan.io/token/0xFFFfFfFfF6E528AD57184579beeE00c5d5e646F0){target=_blank} |
+    |  Statemine   | xcRMRK | [0xffffffFF893264794d9d57E1E0E21E0042aF5A0A](https://moonriver.moonscan.io/token/0xffffffFF893264794d9d57E1E0E21E0042aF5A0A){target=_blank} |
+    |  Statemine   | xcUSDT | [0xFFFFFFfFea09FB06d082fd1275CD48b191cbCD1d](https://moonriver.moonscan.io/token/0xFFFFFFfFea09FB06d082fd1275CD48b191cbCD1d){target=_blank} |
+
+    _*You can check each [Asset ID](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonriver.moonbeam.network#/assets){target=_blank} on Polkadot.js Apps_
```
Comment (eshaben) [inline builders/xcm/xc20/xc20.md:51]: this too

File: builders/xcm/xc20/xc20.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+    |    Kintsugi Alphanet    | xcKINT  | [0xFFFfffff27C019790DFBEE7cB70F5996671B2882](https://moonbase.moonscan.io/token/0xFFFfffff27C019790DFBEE7cB70F5996671B2882){target=_blank} |
+    |    Kintsugi Alphanet    | xckBTC  | [0xFffFfFff5C2Ec77818D0863088929C1106635d26](https://moonbase.moonscan.io/token/0xFffFfFff5C2Ec77818D0863088929C1106635d26){target=_blank} |
+    |    Litentry Alphanet    |  xcLIT  | [0xfffFFfFF31103d490325BB0a8E40eF62e2F614C0](https://moonbase.moonscan.io/token/0xfffFFfFF31103d490325BB0a8E40eF62e2F614C0){target=_blank} |
+    | Parallel Heiko Alphanet |  xcHKO  | [0xffffffFF394054BCDa1902B6A6436840435655a3](https://moonbase.moonscan.io/token/0xffffffFF394054BCDa1902B6A6436840435655a3){target=_blank} |
+    |   Statemine Alphanet    | xcMRMRK | [0xFFffffFfd2aaD7f60626608Fa4a5d34768F7892d](https://moonbase.moonscan.io/token/0xFFffffFfd2aaD7f60626608Fa4a5d34768F7892d){target=_blank} |
+    
+
+     _*You can check each [Asset ID](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/assets){target=_blank} on Polkadot.js Apps_
```
Comment (eshaben) [inline builders/xcm/xc20/xc20.md:71]: and this

File: builders/xcm/xc20/xc20.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+
+## 获取外部XC-20资产的列表 {: #list-xchain-assets }
+
+要获取Moonbase Alpha测试网上目前可用的外部XC-20资产列表，请导向至[Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/explorer){target=_blank}并确保您已连接至Moonbase Alpha。进入页面后，点击**Developer**标签并在下拉菜单中选择**Chain State**。接着，您可以跟随以下步骤查询可用的外部XC-20资产：
+
+1. 在**selected state query**下拉菜单中，选择**assets**
+
+  --8<-- 'text/xc-20/list-of-assets.md'
```
Comment (eshaben) [inline builders/xcm/xc20/xc20.md:79]:
```suggestion
--8<-- 'text/xc-20/list-of-assets.md'
```

File: builders/xcm/xc20/xc20.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+![Fetch list of cross-chain assets](/images/builders/xcm/xc20/xc20/xc20-1.png)
+
+点击按钮后结果将会显现，包含资产以及Moonbase Alpha上已注册的外部XC-20的额外信息。
+
+## 获取外部XC-20资产的元数据 {: #x-chain-assets-metadata }
+
+要快速获得特定XC-20资产的信息，如名称、XC-20表现形式或是资产的位数，您可以使用**metadata**函数以获得其元数据。举例而言，您可以使用资产ID`42259045809535163221576417993425387648`，并跟随以下步骤进行操作：
+
+1. 在**selected state query**下拉菜单中，选择**assets**
+    --8<-- 'text/xc-20/retrieve-metadata.md'
```
Comment (eshaben) [inline builders/xcm/xc20/xc20.md:90]:
```suggestion
--8<-- 'text/xc-20/retrieve-metadata.md'
```

File: builders/xcm/xc20/xtokens.md [hunk 8]
```diff
@@ -24,77 +24,89 @@ description: 学习如何使用X-Tokens pallet将XC-20s传送至其他链。另
...
 
-    |   参数   |     数值      |
-    | :------: | :-----------: |
-    | Version  |      V1       |
-    | Parents  |       1       |
-    | Interior |      X1       |
-    |    X1    |  AccountId32  |
-    | Network  |      Any      |
-    |    Id    | TargetAccount |
+    | Parameter |     Value      |
```
Comment (eshaben) [inline builders/xcm/xc20/xtokens.md:100]:
```suggestion
    | 参数 |     数值      |
```

File: builders/xcm/xc20/xtokens.md [hunk 9]
```diff
@@ -103,44 +115,54 @@ X-Tokens pallet提供以下extrinsics（函数）：
...
 
-4. 要定义XCM资产的多重地点，您需要在Moonbase Alpha上定义中继链上的`UNIT`为源头。每条链皆有不同的定义方法。因此，您需要为各个地点设置不同的资产多重地点。在此例而言，中继链Token将会经由下列函数定义：
+4. 定义XCM资产multilocation，您需要将Moonbase Alpha的中继链中的`UNIT`作为初始点。每条链皆有不同的定义方法。因此，您需要为各个目标设置不同的资产multilocation。在此例而言，中继链Token将会通过以下函数定义：
 
-    |   参数   | 数值 |
-    | :------: | :--: |
-    | Version  |  V1  |
-    | Parents  |  1   |
-    | Interior | Here |
+    | Parameter | Value |
```
Comment (eshaben) [inline builders/xcm/xc20/xtokens.md:134]:
```suggestion
    | 参数 | 数值 |
```

File: builders/xcm/xc20/xtokens.md [hunk 10]
```diff
@@ -103,44 +115,54 @@ X-Tokens pallet提供以下extrinsics（函数）：
...
+    |:---------:|:-----:|
+    |  Version  |  V1   |
+    |  Parents  |   1   |
+    | Interior  | Here  |
+    
+    如果您要将本教程修改成适应可铸造XC-20资产，您需要指定创建资产和资产ID的pallet。因此，您需要设置以下参数：
+
+    |   Parameter    |     Value      |
```
Comment (eshaben) [inline builders/xcm/xc20/xtokens.md:142]:
```suggestion
    |   参数    |     数值      |
```

File: builders/xcm/xc20/xtokens.md [hunk 11]
```diff
@@ -103,44 +115,54 @@ X-Tokens pallet提供以下extrinsics（函数）：
...
 
-    |   参数   |     数值      |
-    | :------: | :-----------: |
-    | Version  |      V1       |
-    | Parents  |       1       |
-    | Interior |      X1       |
-    |    X1    |  AccountId32  |
-    | Network  |      Any      |
-    |    Id    | TargetAccount |
+    | Parameter |     Value      |
```
Comment (eshaben) [inline builders/xcm/xc20/xtokens.md:156]:
```suggestion
    | 参数 |     数值      |
```

File: builders/xcm/xc20/xtokens.md [hunk 12]
```diff
@@ -176,10 +198,17 @@ X-Token预编译合约将会允许开发者通过基于Moonbeam网络的以太
 
 此接口包含以下函数：
 
- - **transfer**(*address* currency_address, *uint256* amount, *Multilocation* *memory* destination, *uint64* weight) —— 用于表示先前提及的`transfer`函数。然而，在使用币种ID之外，您需要提供[XC-20地址](/builders/xcm/xc20/#current-xc20-assets){target=_blank}。多重地点将会以一种特定形式构建，将会在下部分提及
- - **transfer_multiasset**(*Multilocation* *memory* asset, *uint256* amount, *Multilocation* *memory* destination, *uint64* weight) —— 用于表示先前提及的`transferMultiasset`函数。所有多重地点将会以一种特定形式构建，将会在下部分提及
+ - **transfer**(*address* currency_address, *uint256* amount, *Multilocation* *memory* destination, *uint64* weight) —— 用于表示[先前示例](/builders/xcm/xc20/xtokens/#xtokens-transfers-function)中提及的`transfer`函数。然而，在使用币种ID之外，您需要为`currency_address`提供资产预编译地址：
```
Comment (eshaben) [inline builders/xcm/xc20/xtokens.md:201]:
```suggestion
 - **transfer**(*address* currency_address, *uint256* amount, *Multilocation* *memory* destination, *uint64* weight) —— 用于表示[先前示例](/builders/xcm/xc20/xtokens/#xtokens-transfer-function)中提及的`transfer`函数。然而，在使用币种ID之外，您需要为`currency_address`提供资产预编译地址：
```

File: builders/xcm/xc20/xtokens.md [hunk 13]
```diff
@@ -190,15 +219,15 @@ X-Token预编译合约将会允许开发者通过基于Moonbeam网络的以太
 }
 ```
 
-请注意每个多重地点皆有`parents`元素，请使用`uint8`和一组字节定义。Parents代表有多少“hops”在通过中继链中您需要执行的传递向上方向。作为`uint8`，您将会看到以下数值：
+请注意每个multilocation皆有`parents`元素，请使用`uint8`和一组字节定义。Parents代表有多少“hops”在通过中继链中您需要执行的传递向上方向。作为`uint8`，您将会看到以下数值：
 
 |   Origin    | Destination | Parents Value |
```
Comment (eshaben) [inline builders/xcm/xc20/xtokens.md:224]: just noticed this... i think these table headers can be translated

**Review by eshaben — CHANGES_REQUESTED** [review]
this was a big PR... nice job! just some minor things!

**Comment by hyd628** [comment]
Thanks @eshaben for the fixes and feedback, I've pushed a commit for the missing translations.

File: builders/xcm/xc20/mintable-xc20.md [hunk 14]
```diff
@@ -0,0 +1,244 @@
...
+
+- **所有者（Owner）**—— 拥有合约并能够管理资产的账户
+- **创建者（Creator）**—— 负责创建资产并支付相关存入费用的账户
+- **发行者（Issuer）**—— 指定能够发行或是铸造Token的账户，默认为所有者（Owner）
+- **管理员（Admin）**—— 指定能够销毁Token和解锁账户和资产的账户，默认为所有者（Owner）
+- **锁定者（Freezer）**—— 能够锁定账户和资产的指定账户，默认为所有者（Owner）
+
+以下为所有角色各自负责任务的相关表格：
+
+|     **角色**      | **铸造** | **销毁** | **锁定** | **解冻** |
```
Comment (eshaben) [inline builders/xcm/xc20/mintable-xc20.md:30]:
```suggestion
|       角色       |   铸造   |   销毁   |   锁定   |   解冻   |
```

**Pull request merged**